packages/core/content-type-builder/admin/src/components/ComponentCard/index.js

```diff
@@ -24,22 +86,32 @@ function ComponentCard({ component, dzName, index, isActive, isInDevelopmentMode
     schema: { icon: null },
   });
 
+  const onClose = e => {
+    e.stopPropagation();
+    removeComponentFromDynamicZone(dzName, index);
+  };
+
   return (
-    <Wrapper onClick={onClick} className={isActive ? 'active' : ''}>
-      <div>
-        <FontAwesomeIcon icon={icon} />
-      </div>
-      <p>{name}</p>
-      <div
-        className="close-btn"
-        onClick={e => {
-          e.stopPropagation();
-          removeComponentFromDynamicZone(dzName, index);
-        }}
-      >
-        {isInDevelopmentMode && <Close width="7px" height="7px" />}
-      </div>
-    </Wrapper>
+    <button type="button" onClick={onClick}>
+      <ComponentBox className={isActive ? 'active' : ''} borderRadius="borderRadius">
```

Comment:
What is the borderRadius prop for?

Also the Stack expects to receive a `size` props.

Reply:
it's to round up the dynamic comp button